src/shogun/lib/DynamicArray.h

```diff
@@ -621,21 +774,13 @@ template <class T> class CDynamicArray :public CSGObject
 		virtual void init()
 		{
 			set_generic<T>();
+			T* head = m_array.data();
+			m_parameters->add_vector(
+			    &head, &num_elements, "array", "Memory for dynamic array.");
```

Comment:
Hi @vigsterkr and @lisitsyn , what's the relationship between these parameters and `SGObject::clone()`? All these parameters will be clone  into generate a new Object? And serialization ?